### ISSUE TITLE
Unable to run the code with insufficient dataset 

### ISSUE THREAD

**Issue by AnjalyG** [s-omranpour/Fake-News-Detection-Pytorch#1]
0%|          | 0/4232 [00:00<?, ?it/s] --------------------------------------------------------------------------- FileNotFoundError                         Traceback (most recent call last) <ipython-input-7-25555d753c33> in <module>      33 for event in tqdm(eid_train+eid_test):      34     event_str = str(event) ---> 35     tweets = json.load(open(os.path.join(path,event_str+'.json'),"r",encoding='utf-8'))      36     timestamps = []      37     uid = []  FileNotFoundError: [Errno 2] No such file or directory: 'Data/Weibo/3566276356194468.json'

**Comment by s-omranpour** [comment]
Can you explain more about your problem?

**Comment by Dantos7** [comment]
I am having the same problem. The Data folder contains only the files Weibo.txt and Weiboeids. The folder Weibo and its content is not present in this repository. This causes the FileNotFoundError mentioned by AnjalyG

**Comment by crushr** [comment]
yep,I got the same error. I guess it's because it can not load the json data. But I don't know how to fix it. the tqdm's progress bar is 0 .here is error:
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-13-b4cab195bebf> in <module>
     33 for event in tqdm(eid_train+eid_test):
     34     event_str = str(event)
---> 35     tweets = json.load(open(os.path.join(path,event_str+'.json'),"r"))
     36     timestamps = []
     37     uid = []

FileNotFoundError: [Errno 2] No such file or directory: './Data/Weibo/3566276356194468.json'
